ansible_ai_connect/main/settings/base.py

```diff
@@ -36,52 +36,50 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
 
-ANSIBLE_AI_MODEL_MESH_HOST = os.getenv(
-    "ANSIBLE_AI_MODEL_MESH_HOST", "https://model.wisdom.testing.ansible.com"
-)
-ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT = os.getenv("ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT", 443)
-ANSIBLE_AI_MODEL_MESH_INFERENCE_URL = (
-    f"{ANSIBLE_AI_MODEL_MESH_HOST}:{ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT}"
-)
-
+# ==========================================
+# Model Provider
+# ------------------------------------------
 t_model_mesh_api_type = Literal[
     "grpc", "http", "dummy", "wca", "wca-onprem", "wca-dummy", "ollama", "llamacpp", "bam"
 ]
 ANSIBLE_AI_MODEL_MESH_API_TYPE: t_model_mesh_api_type = os.getenv(
     "ANSIBLE_AI_MODEL_MESH_API_TYPE", cast(t_model_mesh_api_type, "http")
 )
-t_model_mesh_healthcheck_protocol = Literal["http", "https"]
-ANSIBLE_AI_MODEL_MESH_API_HEALTHCHECK_PROTOCOL: t_model_mesh_healthcheck_protocol = os.getenv(
-    "ANSIBLE_AI_MODEL_MESH_API_HEALTHCHECK_PROTOCOL",
-    cast(t_model_mesh_healthcheck_protocol, "https"),
+
+t_model_mesh_protocol = Literal["http", "https"]
+ANSIBLE_AI_MODEL_MESH_PROTOCOL: t_model_mesh_protocol = os.getenv(
+    "ANSIBLE_AI_MODEL_MESH_PROTOCOL",
+    cast(t_model_mesh_protocol, "https"),
 )
-ANSIBLE_AI_MODEL_MESH_API_HEALTHCHECK_PORT = (
-    ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT
-    if ANSIBLE_AI_MODEL_MESH_API_TYPE == "http"
-    else (
-        os.getenv("ANSIBLE_AI_MODEL_MESH_API_HEALTHCHECK_PORT", "8443")
-        if ANSIBLE_AI_MODEL_MESH_API_TYPE == "grpc"
-        else None
-    )
+ANSIBLE_AI_MODEL_MESH_HOST = os.getenv(
+    "ANSIBLE_AI_MODEL_MESH_HOST", "https://model.wisdom.testing.ansible.com"
```

Comment:
It should be:

```python
ANSIBLE_AI_MODEL_MESH_HOST = os.getenv("ANSIBLE_AI_MODEL_MESH_HOST") or "https://model.wisdom.testing.ansible.com"
```

Reply:
Yes, yes. I know.. this PR has a merge conflict with the changes you made and I am working on it.